sass/styles/chart.scss

```diff
@@ -221,7 +221,7 @@ html, body {
     padding: 0px;
     margin-bottom: 17px;
     border-radius: 4px;
-    z-index: 12;
+    z-index: 15;
```

Comment:
`15` is the minimum value that works? Could we use a variable with a good name for this?

Reply:
@salar-fs  to be honest, we don't have exact valid range for that. It comes to `15` as 12 seems to have a conflict with barrier shade. that would be a good idea to have constant numbers for them, we should handle it in refactoring